frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -844,7 +844,8 @@ function ReaderPaging:onGotoPageRel(diff)
     logger.dbg("goto relative page:", diff)
     local new_va = self.visible_area:copy()
     local x_pan_off, y_pan_off = 0, 0
-    local right_to_left = (self.ui.document.configurable.writing_direction > 0)
+    local right_to_left = self.ui.document.configurable.writing_direction
+                        and self.ui.document.configurable.writing_direction > 0
```

Comment:
Alignment would look nicer like this. But no objection to sticking it on one line on my part, probably clearer if anything.
```suggestion
                          and self.ui.document.configurable.writing_direction > 0
```

Reply:
Even though it makes the line longer than 120 characters (usual limit in Lua) ?

Reply:
A line that's more than some 100-120 characters is a rough indicator that there might be too much nested complexity. Other than that line length has no particular significance.

Reply:
OK. Till now, I thought the main reason for that was to avoid lines getting out the screen in editors.